Installation/CHANGES.md

```diff
@@ -62,6 +62,12 @@ Release date: March 2019
     `Arr_polyline_traits_2`, `Arr_polycurve_traits_2`, and
     `Arr_polycurve_basic_traits_2`.
 
+### 2D and 3D Mesh Generation
+-   Added 3 functions for writing in XML VTK formats:
+    - `CGAL::output_to_vtu()`
+    - `CGAL::write_VTU()`
+    - `CGAL::write_VTP()`
```

Comment:
This modification is wrong because `write_VTP` is added in the BGL package. I also think that the changelog should give a short sentence about the usage of each function.